src/module/Mailman/src/Mailman/MailmanWorker.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * This file is part of Athene2.
+ *
+ * Copyright (c) 2013-2019 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2013-2019 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/athene2 for the canonical source repository
+ */
+
+namespace Mailman;
+
+use ClassResolver\ClassResolverAwareTrait;
+use Common\Traits\ObjectManagerAwareTrait;
+use Notification\Entity\NotificationInterface;
+use Zend\EventManager\EventManagerAwareTrait;
+
+class MailmanWorker
```

Comment:
On second thought: I'd move that to the `Notification` module, i.e.
- `Notification` is responsible for creating the notifications and handling the delivery method (e.g. sending serlo.org notifications directly, batch out emails)
- `Mailman` is responsible for sending out emails